server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Compactor.java

```diff
@@ -366,8 +366,8 @@ else if (env.getIteratorScope() == IteratorScope.minc)
       else
         throw new IllegalArgumentException();
 
-      SortedKeyValueIterator<Key,Value> itr = iterEnv.getTopLevelIterator(IteratorUtil
-          .loadIterators(env.getIteratorScope(), cfsi, extent, acuTableConf, iterators, iterEnv));
+      SortedKeyValueIterator<Key,Value> itr = iterEnv.getTopLevelIterator(IterConfigUtil
+          .loadIterators(env.getIteratorScope(), cfsi, acuTableConf, iterators, iterEnv));
```

Comment:
The extent was not carried forward, is it no longer needed?

Reply:
It wasn't used anymore.  I am not sure it ever was either.